frontend/src/components/panels/LayerTree.vue

```diff
@@ -425,18 +423,17 @@ export default defineComponent({
 			recurse(expandData, this.layers, this.layerCache);
 		});
 
-		registerResponseHandler(ResponseType.UpdateLayer, (responseData) => {
-			const updateData = responseData as UpdateLayer;
-			if (updateData) {
-				const responsePath = updateData.path;
-				const responseLayer = updateData.data;
-
-				const layer = this.layerCache.get(responsePath.toString());
-				if (layer) Object.assign(this.layerCache.get(responsePath.toString()), responseLayer);
-				else this.layerCache.set(responsePath.toString(), responseLayer);
-				this.setBlendModeForSelectedLayers();
-				this.setOpacityForSelectedLayers();
+		registerJsMessageHandler(UpdateLayer, (updateData) => {
+			const responsePath = updateData.data.path;
+			const responseLayer = updateData.data;
+
+			const layer = this.layerCache.get(responsePath.toString());
+			if (layer) Object.assign(this.layerCache.get(responsePath.toString()), responseLayer);
+			else {
+				this.layerCache.set(responsePath.toString(), responseLayer);
 			}
```

Comment:
Did the autoformatter do this? I try to keep consistency with the if and else cases, so they either both have the curlies or both don't. If you have control over it, and the autoformatter doesn't change it back, could you switch it so both cases either do or don't have them? (They are arguably kinda long lines, so I don't have a strong preference either way if we use or omit curlies.)

Reply:
Im an always curly bracer :)